Airport/Program.cs

```diff
@@ -0,0 +1,577 @@
+﻿
+using System;
+using System.Threading;
+using System.Text.Json;
+using System.Linq;
+using System.Xml;
+
+
+namespace Airport
+{
+    public class Program
+    {
+        readonly static string path = @"Json\";
+        readonly static string jsonPath = Path.Combine(Directory.GetParent(Directory.GetCurrentDirectory()).Parent.Parent.FullName, path);
+        public static int id = 0;
+        //Deserialize Flight
+        readonly static string filenameFlights = $"{jsonPath}flights.json";
+        readonly static string jsonTextFlights = File.ReadAllText(filenameFlights);
+        readonly static List<Flights> flights = JsonSerializer.Deserialize<List<Flights>>(jsonTextFlights);
+        //Deserialize Passenger
+        readonly static string filePassengers = $"{jsonPath}passenger.json";
+        readonly static string jsonTextPass = File.ReadAllText(filePassengers);
+        readonly static List<Passengers> passengers = JsonSerializer.Deserialize<List<Passengers>>(jsonTextPass);
+        //Deserialize AirCarrier
+        readonly static string filename1 = $"{jsonPath}airCarriers.json";
+        readonly static string jsonTextAir = File.ReadAllText(filename1);
+        readonly static List<AirCarrier> airCarriers = JsonSerializer.Deserialize<List<AirCarrier>>(jsonTextAir);
+
+
+
+        static void Main(string[] args)
+        {
+            try
+            {
+                InitialMenu();
+            }
+            catch (IndexOutOfRangeException ex)
+            {
+                ExceptionMethod(ex);
+            }
+            catch (FormatException ex)
+            {
+                ExceptionMethod(ex);
+            }
+        }
+
+        public static void ExceptionMethod(Exception e)
+        {
+            Console.WriteLine(e.Message);
+            Console.WriteLine(e.StackTrace);
+        }
+
+        public static void InitialMenu()
+        {
+            bool done = default;
+            while (!done)
+            {
+                Console.Clear();
+                WriteLogo();
+                MenuItems(1, "Login");
+                MenuItems(2, "Check buy a ticket");
+                MenuItems(3, "Check booking status");
+                MenuItems(4, "Logout");
+                var optionInit = int.Parse(Console.ReadLine());
+                switch (optionInit)
+                {
+                    case 1:
+                        Login();
+                        break;
+                    case 2:
+                        Buy();
+                        break;
+                    case 3:
+                        CheckBookingStatus();
+                        break;
+                    case 4:
+                        done = true;
+                        break;
+                    default:
+                        Console.WriteLine("Enter valid option!");
+                        Thread.Sleep(1000);
```

Comment:
for what? 
Why it should be avoided

It doesn't release the lock
It doesn't gurantee that the execution will start after sleeping time (So it may keep waiting forever - obviously a rare case)
If we mistakenly put a foreground processing thread on sleep then we wouldn't be able to close that application till x milliseconds.
We now full loaded with new concurrency package for specific problems (like design patterns (ofcourse not exactly), why to use Thread.sleep(x) then.